python3_user_features.rst

```diff
@@ -404,3 +404,30 @@ symbols as variable names, e.g.:
 
 Use caution if you're planning to share your code, though, as it's fairly easy
 to produce illegible code this way.
+
+More useful exceptions
```

Comment:
I'd either add a couple more examples or mention them in passing at the end of the section, possibly with a link.